client/MentorPanel/src/components/video.js

```diff
@@ -55,25 +55,15 @@ const VideoPlayer = ({ width }) => {
     )
 }
 
-class Video extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = { width: 0 };
-    }
-
-    componentDidMount() {
-        const width = Math.min(window.innerWidth, 500)
-        this.setState({ width })
-    }
+const Video = () => {
+    const height = (window.innerHeight * 0.5) - 90
```

Comment:
same here about height: can it be a param to make this component for reusable among layouts?